regcomp.h

```diff
@@ -112,6 +112,42 @@ typedef struct regexp_internal {
 
 /* this is where the old regcomp.h started */
 
+
```

Comment:
In the commit message: `s/64 aligned/64 bit aligned/`; `s/that is stored/that it is stored/`; `s/which part/which is part/`

Reply:
Ah. I guess I will do that in a fixup before final merge so I dont clobber this feedback with a rebase :-)

Reply:
Please don't resolve this until I squash the patches.